spec/open_api_parser/pointer_spec.rb

```diff
@@ -33,27 +33,29 @@
 
       resolutions.each do |pointer, expected|
         expect(OpenApiParser::Pointer.new(pointer).resolve(DOCUMENT)).to eq(expected)
+        expect(OpenApiParser::Pointer.new('#' + pointer).resolve(DOCUMENT)).to eq(expected)
       end
     end
 
     it "works with escaped RFC examples" do
       resolutions = {
-        "#" => DOCUMENT,
-        "#/foo" => ["bar", "baz"],
-        "#/foo/0" => "bar",
-        "#/" => 0,
-        "#/a~1b" => 1,
-        "#/c%25d" => 2,
-        "#/e%5Ef" => 3,
-        "#/g%7Ch" => 4,
-        "#/i%5Cj" => 5,
-        "#/k%22l" => 6,
-        "#/%20" => 7,
-        "#/m~0n" => 8,
```

Comment:
These example are directly from the JSON Pointer RFC. We need to continue to support them.

Reply:
👍 Change the spec so that it tests by dropping the pound sign, not adding back: 1ea0e734bd772991e75c229987b3556929add579